conanfile.py

```diff
@@ -157,6 +170,11 @@ def config_options(self):
             self.options.remove('video_rpi')
         if self.settings.os != "Windows":
             self.options.remove("directx")
+            self.options.remove("render_d3d")
+
+        if self.settings.os != "Macos": # iOS/tvOS/... ?
```

Comment:
```suggestion
        if not tools.is_apple_os():
```
https://docs.conan.io/en/latest/reference/tools.html#tools-is-apple-os